security/appsec/intro/what-else-do-you-need-to-know.md

```diff
@@ -68,3 +62,11 @@ How memory in a computer works, and where data is located at any given time
 * Binary
 * Hexadecimal
 * Cryptography
+
+---
+## Footnotes
+
+[1:Levels of AppSec]
+It's something everyone needs to know at some level, so that when major outbreaks occur they can take steps to protect themselves and the applications they develop
+
+At more advanced levels, it means strictly following regulatory guidelines to ensure certain kinds of data are very well protected. At very advanced levels, it can mean research, penetration testing, and more.
```

Comment:
```suggestion
At more advanced levels, it means strictly following regulatory guidelines to ensure certain kinds of data are very well protected. 

At very advanced levels, it can mean research, penetration testing, and more.
```